### PR TITLE
Add support for watching multiple Nginx instances

### LINKED ISSUE #1: Monitoring multiple Nginx instances
It's definitely a need to monitor multiple Nginx instances at a time just like to approach mentioned at https://github.com/elastic/libbeat/issues/37#issuecomment-152861180.

The config file would define a list of servers to be monitored  (each of them may use different formats, i.e. stub v.s. nginx plus; though, let's not get to that complicated things first :tongue:). Nginxbeat would monitor each of them in a tick.

### PR DESCRIPTION
**Pull request by mrkschan**
Resolve #1.

### CODE REVIEW

**Comment by mrkschan** [comment]
Hey @ruflin, would you mind to take a look at https://github.com/mrkschan/nginxbeat/pull/9/files#diff-edf2363e464e98a98f11ccb6671e1d6dR98? Am I going to the right direction on watching multiple Nginx instances?

File: nginxbeat.go [hunk 1]
```diff
@@ -90,46 +95,52 @@ func (nb *Nginxbeat) Setup(b *beat.Beat) error {
...
+				p = parser.NewPlusParser()
+			}
+
+			ticker := time.NewTicker(nb.period)
+			defer ticker.Stop()
+
+			for {
+				select {
+				case <-nb.done:
+					goto GotoFinish
```
Comment (ruflin) [inline nginxbeat.go]: why do you use goto and not just break?

**Comment by ruflin** [comment]
@mrkschan Just had a quick look but I think it goes in the right direction. Does it work as expected or do you have any issues?

**Comment by mrkschan** [comment]
It's working though i just wanna check if using goroutine like that is a good practice. thx for taking a look, @ruflin :)

File: nginxbeat.go (hunk 1 above)
Comment (mrkschan) [inline nginxbeat.go]: I think this is just a matter of preferences. We can have `for condition {}` or `break` or even `return` there. Is there any preferences in the libbeat team? I can stick with that for consistency.

Comment (ruflin) [inline nginxbeat.go]: @mrkschan I think we have the same preference issue in libbeat so there is no standard ;-)

Comment (mrkschan) [inline nginxbeat.go]: @ruflin so i will wait until there is one from Elastic. i will keep `goto` right now for the sake of consistency inside that func(). thx for the info :)

**Pull request merged**